src/test/java/org/sagebionetworks/web/unitclient/widget/profile/UserProfileEditorWidgetImplTest.java

```diff
@@ -239,9 +239,23 @@ public void testConfigureNoEmail() {
   @Test
   public void testConfigureNotOwner() {
     when(mockAuthController.getCurrentUserPrincipalId()).thenReturn(null); //anonymous
```

Comment:
If not the owner and not ACT, then emails should not be set (by the service), and view.setEmails() should not be called.